modules/_labs/header/react/lib/Header.tsx

```diff
@@ -201,7 +202,7 @@ const ChildrenSlot = styled('div')<Pick<HeaderProps, 'centeredNav' | 'themeColor
     '> *:last-child': {
       marginRight: isCollapsed ? '' : 0,
     },
-    flexGrow: !isCollapsed && centeredNav ? 1 : 'unset',
+    flex: !isCollapsed && centeredNav ? `1 0 auto` : 'unset',
```

Comment:
Is this line doing anything, when I was testing it didn't seem to effect the centering of the menu.

Reply:
This allows the nav to be centered in your new example in #422, without it, the inner `nav` element would look like this:

<img width="1603" alt="Screen Shot 2020-01-27 at 8 21 23 AM" src="https://user-images.githubusercontent.com/14299381/73192555-0b9d5880-40de-11ea-9c3c-28909ddb706a.png">